README.md

```diff
@@ -1,2 +1,12 @@
 # stock-simulator
 Application simulating a stock data feed.
+
+
+## Running the Program
+
+1. `mkdir build`
```

Comment:
this shouldnt be in the steps to do it, should autogenerate from the make or cmake or include it

Reply:
This was meant as a very preliminary README just to have this here. However I agree that this is essential and I will add the Makefile for this in this PR already.

Reply:
I created a Makefile that executes the order of events when running `make`.